### DISCUSSION TITLE
Push only for Nextcloud?

### DISCUSSION THREAD

**Discussion by Offerel** [bitfireAT/webdav-push#15] [General]
Is this Push implementation only meant to work with Nextcloud? Personally, I'm not the biggest fan of Nextcloud, but I'm using SabreDAV with some patches. I'm very interested to have this also working with standard SabreDAV. I had a discussion with evert some years ago about the same topic, but we came to no real starting point.

Hopefully this will change now.

**Comment by rfc2822** [comment]
No, Push is intended as an open standard that can be used with every server. We only plan to make it working with Nextcloud as kind of a sample implementation.

**Reply by Offerel** [reply]
If you need testers for SabreDAV, please let me know. I can also try to implement some patches and push them to the SabreDAV project. Im not a professional developer, but maybe that what i know is skilled enough to create a working example. From my side of view, we can also discuss this in german, at least it would be easier for me. Im very interested in this topic and would like to help as much as i can.
